### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by JeromeGalan**
Fix a deadlink in Readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Feel free to use these examples as a starting point for your own projects, and d
 
 ### Apps folder
 
-This folder contains the container's sources of each **App** project (See [demo boards tutorials](https://docs.luos.io/pages/tutorials/tutorials.html?highlight=servo#demo-boards-tutorials) to try some apps). An App is the intelligence of your project, and the examples provided will show you how to set up an App for your project.
+This folder contains the container's sources of each **App** project (See [demo boards tutorials](https://docs.luos.io/tutorials/demo-boards/servomotor) to try some apps). An App is the intelligence of your project, and the examples provided will show you how to set up an App for your project.
```
Comment (BiliouriV) [inline README.md:19]: it is called service and not container now...

**Review by BiliouriV — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (JeromeGalan) [reply README.md]: Good !

It seems that some containers are still hidden in the Readme files.
I'll check the other Readme in all Luos repositories.

The correction is done on current PR

**Review by BiliouriV — COMMENTED** [review]
✅

**Review by BiliouriV — APPROVED** [review]
✅

**Pull request merged**